app/controllers/snps_controller.rb

```diff
@@ -4,6 +4,8 @@ class SnpsController < ApplicationController
   before_filter :find_snp, except: [:index, :json,:json_annotation]
 
   def index
+    params.delete(:sort)
+    params.delete(:direction)
     @snps = Snp.order(sort_column + ' ' + sort_direction)
```

Comment:
Wouldn't it have been enough to set `Snp.order(ranking: :desk)`, [analogous to this](https://apidock.com/rails/ActiveRecord/QueryMethods/order)? Then the additional parameters wouldn't matter at all?